src/lib/CombatCalc.ts

```diff
@@ -45,7 +45,7 @@ import { keys } from '@/utils';
 const DEFAULT_ATTACK_SPEED = 4;
 const SECONDS_PER_TICK = 0.6;
 
-const TTK_DIST_MAX_ITER_ROUNDS = 1000;
+const TTK_DIST_MAX_ITER_ROUNDS = 4000;
```

Comment:
This is far less than the original for low-speed weapons. Let's use `TTK_DIST_MAX_ITER_ROUNDS * weaponSpeed` as the upper bound instead and keep this at 1000.